jsonnet/kube-prometheus/ksm-autoscaler/ksm-autoscaler.libsonnet

```diff
@@ -1,13 +1,11 @@
-local k = import 'github.com/ksonnet/ksonnet-lib/ksonnet.beta.4/k.libsonnet';
-
 {
   _config+:: {
     versions+:: {
-      clusterVerticalAutoscaler: "v0.8.1"
+      clusterVerticalAutoscaler: 'v0.8.1',
```

Comment:
Should we change it to `0.8.1` and add `v` prefix where we are composing a full image name? This way this var can be used as a correct `app.kubernetes.io/version` label value.